<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlutoControlTesterWindow</class>
 <widget class="QMainWindow" name="PlutoControlTesterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>452</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>452</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PLUTO Test Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gbControlType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Choose Control Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QRadioButton" name="radioPosition">
            <property name="font">
             <font>
              <family>Bahnschrift Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioNone">
            <property name="font">
             <font>
              <family>Bahnschrift Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>No Control</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioTorque">
            <property name="font">
             <font>
              <family>Bahnschrift Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Torque</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="radioPositionLinear">
            <property name="font">
             <font>
              <family>Bahnschrift Light</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Position Linear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="dsbTgtDur">
         <property name="font">
          <font>
           <family>Cascadia Mono Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTargetDuration">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Target Duration (s):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblFeedforwardTorqueValue">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Feedforward Torque Value (Nm):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dsbTorqTgtValue">
         <property name="font">
          <font>
           <family>Cascadia Mono Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblPositionTargetValue">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Target Position Value (deg):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="dsbPosTgtValue">
         <property name="font">
          <font>
           <family>Cascadia Mono Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblControlBoundValue">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Control Bound:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="dsbCtrlBndValue">
         <property name="font">
          <font>
           <family>Cascadia Mono Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblControlGainValue">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Control Gain:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="dsbCtrlGainValue">
         <property name="font">
          <font>
           <family>Cascadia Mono Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetTarget">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Set Target</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbCtrlHold">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Control Hold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCtrlDecay">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Control Decay</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
